components/ArticleItem.js

```diff
@@ -56,16 +71,6 @@ export default function ArticleItem({
       </Link>
 
       <style jsx>{listItemStyle}</style>
-      <style jsx>{`
-        .latest-reply {
-          background-color: #64b5f6;
-          padding: 1rem;
-          border-radius: 4px;
-        }
-        .item:hover .latest-reply {
```

Comment:
@yanglin5689446 Since it's difficult to have cross-component style, and considering that we are revamping UI very soon, I did not migrate this style.